openpathsampling/experimental/storage/monkey_patches.py

```diff
@@ -64,7 +64,14 @@ def inner(cls, dct):
     paths
 ]
 
+_IS_PATCHED_SAVING = False
+_IS_PATCHED_LOADING = False
```

Comment:
For global variables, I would like to have an indication where they come from (and slightly more protection against a separate package doing the same thing), do you mind changing these (and all the other ones) to
```suggestion
_OPS_IS_PATCHED_SAVING = False
_OPS_IS_PATCHED_LOADING = False
```

Reply:
globals aren't true globals -- they're still module scope (even if the module they're scoped to is `__main__`). [The `global` keyword is necessary to set module-level globals from within a function.](https://stackoverflow.com/a/7060755) So I don't think this is a problem.

Reply:
you are right, sorry about my confusion.